content/docs/command-reference/metrics/index.md

```diff
@@ -27,21 +21,76 @@ positional arguments:
 In order to track metrics associated to machine learning experiments, DVC has
 the ability to mark a certain stage <abbr>outputs</abbr> as files containing
 metrics to track (see the `--metrics` option of `dvc run`). Metrics are
-project-specific floating-point values e.g. `AUC`, `ROC`, etc.
-
-Supported file formats: JSON. Metrics can be organized in a tree hierarchy in a
-JSON file. DVC addresses the metrics by the tree path.
+project-specific floating-point or integer values e.g. `AUC`, `ROC`,
+`False-Positives` etc.
+
+[Show](/doc/command-reference/metrics/show), and
+[diff](/doc/command-reference/metrics/diff), commands are available to manage
+<abbr>DVC project</abbr> metrics.
+
+## Formats and metrics types
+
+Supported file formats: JSON and YAML. Metrics can be organized in a tree
+hierarchy in a JSON file or a YAML file. DVC addresses the metrics by the tree
+path. In the example below five metrics are presented: `train.accuracy`,
+`train.loss`, `train.TN`, `train.FP` and `time_real`.
+
+```json
+{
+  "train": {
+    "accuracy": 0.9886999726295471,
+    "loss": 0.041855331510305405,
+    "TN": 473,
+    "FP": 845
+  },
+  "time_real": 344.61309599876404
+}
+```
 
 DVC itself does not ascribe any specific meaning for these numbers. Usually
 these numbers are produced by the model training or model evaluation code and
 serve as a way to compare and pick the best performing experiment.
 
-[Add](/doc/command-reference/metrics/add),
-[show](/doc/command-reference/metrics/show),
-[diff](/doc/command-reference/metrics/diff),
-[modify](/doc/command-reference/metrics/modify), and
-[remove](/doc/command-reference/metrics/remove) commands are available to set up
-and manage <abbr>DVC project</abbr> metrics.
+## Default metrics files
+
+`dvc metrics show` and `dvc metrics diff` commands by default use all metrics
+files that are specified in `dvc.yaml`. So, there no need to specify metrics
+file name to see the deault metrics. The metrics files can be added to
+`dvc.yaml` by options `--metrics` (`-m`) or `--metrics-no-cache` (`-M`) of
+`dvc run` command or manualy to `metrics` section of a stage:
+
+```yaml
+stages:
+  train:
+    cmd: python train.py
+    deps:
+      - users.csv
+    params:
+      - epochs
+      - dropout
+      - lr
+    outs:
+      - model.pkl
+    metrics:
+      - summary.json:
+          cache: false
+```
+
+`False` value of the additional `cache` option specifies that this is not data
+file, it should not be moved to DVC cache and it should be commited to Git
+history instead.
```

Comment:
```suggestion
`cache: false` above specifies that `summary.json is not a data file: it will not
be <abbr>cached</abbr> by DVC. Metric files are normally committed with
Git instead.
```